### ISSUE TITLE
Allow to switch tabs without the UI picker (or just switch tabs immediately when the UI picker is open)

### ISSUE THREAD

**Issue by evandrocoan** [microsoft/vscode#132825]
As explained on https://github.com/microsoft/vscode/issues/90465#issuecomment-588058257, 
> to clarify, this kind of navigation is possible in VSCode and Ctrl+Tab is even assigned as default keybinding. What is not working is using this without a picker UI.

Now to switch tabs with `Ctrl+Tab`, a pop up show up and it only shows the contents when the view/editor is picked on the pop up. This is quite annoying because I would like to see the full view before selecting (by releasing the Ctrl key as already done by Sublime Text which does not have the picker pop up).

![](https://user-images.githubusercontent.com/900690/74811827-9692fc80-52f2-11ea-8428-ff60daa3d5ec.gif)

Related to: https://github.com/microsoft/vscode/issues/132823 - Do not hide tabs view when showing command palette, quickpanel or alt+tab views (show it a few inches below instead)

This was already asked on https://stackoverflow.com/questions/38957302/is-there-a-quick-change-tabs-function-in-visual-studio-code, but the proposed solution is to use the next/previous editor/tab/view command which do not accept/respect MRU (Most Recently Used). As having MRU is vital for a good programing environment, using next/previous tab button is not an acceptable solution.

**Comment by bpasero** [comment]
There is a ton of commands that allow you do to that:

![image](https://user-images.githubusercontent.com/900690/132804714-37d69061-0232-4601-a927-740ae4a87c33.png)

Did you try? Start with "Open next recently used editor" and "Open previously used editor"

**Comment by evandrocoan** [comment]
I tried Open Previous Recently Used Editor and some others, but they are not MRU as the default `Ctrl+Tab` key binding is. If I press it once it moves to my last tab (after I release the Ctrl/tab keys). But if I press it again, it does not move back to my most recently used tab, it keeps moving backward or forwarding in tab history.

I just would something like the current `Ctrl+Tab` to show the picker and "switch" immediately (show the full view/editor) instead of waiting for the `ctrl` key to be released. This looks the same as the request https://github.com/microsoft/vscode/issues/90465. This is also how Sublime Text behaves by default with his `Ctrl+Tab` key.

**Comment by evandrocoan** [comment]
> I don't really understand how that would work, you could then essentially only navigate to the previous editor and nothing else?

It would work exactly as the default `Ctrl+Tab` does. The only difference is that the Ui Picker pop-up would not show up. Instead, it would immediately "switch" tabs, but it would only commit an entry to the MRU history when I released the Ctrl key. For example:
1. I have Editor 1, Editor 2, and Editor 3 open, my MRU stack would be (Editor 1, Editor 2), i.e., now the active Editor is Editor 3, because I first opened Editor 1, then opened Editor 2, and finally opened Editor 3.
2. Now, I press `Ctrl`, press `Tab`, release `Tab`, press `Tab`, release `Tab`, release `Ctrl`. Now the selected Editor is Editor 1, with an MRU stack (Editor 2, Editor 3).
3. So, if now I press `Ctrl`, press `Tab`, release `Tab`, release `Ctrl`. Now the selected Editor is Editor 3 again (the top of the stack) and the MRU stack would be (Editor 2, Editor 1).
4. If I do the same key combination (with single `Tab` keystroke), i.e., press `Ctrl`, press `Tab`, release `Tab`, release `Ctrl`. Now the selected Editor is Editor 1 again (the top of the stack) and the MRU stack would be (Editor 2, Editor 3).
5. And if I do now 2 tabs press in a row, i.e., press `Ctrl`, press `Tab`, release `Tab`, press `Tab`, release `Tab`, release `Ctrl`. Now the selected Editor would be Editor 2, with the MRU stack (Editor 3, Editor 1)

My problem with the current method is that I do not see the Editor's contents until I commit the Ui Picker. I would take a much more reliable decision of which Editor to choose if I can see its contents before commit the Ui Picker (and changing the MRU stack). As it is now, I have to commit the Editor just by knowing its file name, which is quite difficult to remember, especially if I have 2 or more Editors on the same buffer (i.e., the same file name).

For example, the default Windows 10 `Alt+Tab` has the same behavior as the VSCode Ctrl+Tab, but with Windows 10 `Alt+Tab`, I can see a preview (miniature picture) of the window, before committing/switching to it. With VSCode now, I do not have such Editor Miniature Picture of the editor. And I would not ask for a miniature picture of the editor because I think it is more reliable just to immediately show the contents and only commit to the last viewed (and now active) editor, when I release the `Ctrl` key.

In regard of Miniature Picture, I have opened a Issue against Sublime Text core (https://github.com/sublimehq/sublime_text/issues/1789) to show a preview of the view stack while Ctrl on Ctrl+Tab is held down, as the default behavior of Sublime Text is the one I am describing on the steps above. But I am not sure if it would be nice to have something on top of my Editor when I am switching (and previewing) the MRU stack. So, I would rather like to have this Miniature Picture feature, out of VSCode and Sublime Text for now.
